src/components/timing/TimingControls.vue

```diff
@@ -142,7 +190,8 @@ const handleStop = async () => {
       >
         {{ capitalizedProjectNames }}
         <v-sheet width="150">
-          <h3 class="text-primary">{{ capitalizeLetters(dailyHour.data.project) }}</h3>
+          <!--<h3 class="text-primary">{{ capitalizeLetters(dailyHour.data.project) }}</h3>-->
+          <!--Capitalize letters funcion is throwing error -->
```

Comment:
The function is throwing error because you are sending wrong data on postDailyHours (line 46)

Reply:
funtion working fine :heavy_check_mark: